prow/config.yaml

```diff
@@ -184,6 +184,148 @@ presubmits:
     context: pull-kops-e2e-kubernetes-aws
     rerun_command: "/test pull-kops-e2e-kubernetes-aws"
     trigger: "(?m)^/test( all| pull-kops-e2e-kubernetes-aws),?(\\s+|$)"
+  kubernetes/kubernetes-anywhere:
+  - name: pull-kubernetes-e2e-kubeadm-gce
```

Comment:
This should probably have a different name from the one run in k/k I think. Perhaps `pull-kubernetes-anywhere-e2e-kubeadm-gce`.

Reply:
The rest of the config is in `jobs/config.json`, the k/k presubmit uses a shared build from `pull-kubernetes-bazel-build` on the same PR, you probably want to use a prebuilt copy of kubernetes somewhere and build kubeadm instead,